docs/cms.rst

```diff
@@ -24,58 +24,326 @@ www.mozilla.org.
+  be defined in ``/bedrock/cms/``.
+
+Structuring code in this way should hopefully help to keep things organized and
+migrations in a manageable state.
+
+Creating a new page model
+-------------------------
+
+Let’s start by creating a new Wagtail page model called ``TestPage``
+in ``bedrock/mozorg/models.py``.
+
+.. code-block:: python
+
+    from django.db import models
+
+    from wagtail.admin.panels import FieldPanel
+    from wagtail.fields import RichTextField
+
+    from bedrock.cms.models.base import AbstractBedrockCMSPage
+
+    class TestPage(AbstractBedrockCMSPage):
+        heading = models.CharField(max_length=255, blank=True)
+        body = RichTextField(
+            blank=True,
+            features=settings.WAGTAIL_RICHTEXT_FEATURES_FULL,
+        )
+
+        content_panels = AbstractBedrockCMSPage.content_panels + [
+            FieldPanel("heading"),
+            FieldPanel("body"),
+        ]
+
+        template = "mozorg/test_page.html"
+
+Some key things to note here:
+
+- ``TestPage`` is a subclass of ``AbstractBedrockCMSPage``, which is a common base
+  class for all Wagtail pages in bedrock. Inheriting from ``AbstractBedrockCMSPage``
+  allows CMS pages to use features that exist outside of Wagtail, such as rendering
+  Fluent strings and other L10n methods.
+- The ``TestPage`` model defines two database field called ``heading`` and ``body``.
+  The ``heading`` field is a ``CharField`` (the most simple text entry field type),
+  and ``body`` is a ``RichTextField``. The HTML tags and elements that a content
+  editor can enter into a rich text field are defined in
+  ``settings.WAGTAIL_RICHTEXT_FEATURES_FULL``.
+- Both fields are added to the CSM admin panel by adding each as a ``FieldPanel`` to
+  ``content_panels``. If you forget to do this, that's usually why you don't see the
+  field in the CMS admin.
+- Finally, the template used to render the page type can be found at
+  ``mozorg/test_page.html``.
+
+Running migrations
+------------------
 
-Details to come
+Once you have your model defined, it’s then time to run migrations to take that
+model and create a database table:
 
+.. code-block:: shell
 
-Editing current content surfaces
---------------------------------
+    ./manage.py makemigrations
 
-Official `Editor Guide`_.
+You can then run migrations using:
 
-Bedrock-specific details to come.
+.. code-block:: shell
 
+    ./manage.py migrate
 
+Many times when you make changes to a model, it will also mean that the structure of
+the database table has changed. So as a general rule it’s good to form a habit of
+running the above steps after making changes to your model. Each migration you make
+will add a new migration file to the ``/migrations`` directory. When doing local
+development for a new page you might find yourself doing this several times, so to help
+reduce the number of migration files you create you can also squash / merge them.
```

Comment:
Related: we can/will automate this to help avoid footguns https://github.com/mozilla/bedrock/issues/14859